app/src/main/java/com/mediaan/mymediaan/view/ftu/CreateProfileScreen.kt

```diff
@@ -0,0 +1,305 @@
+                modifier = Modifier.fillMaxWidth(),
+                supportingText = {
+                    Text(
+                        text = "min $minAge y/o",
+                        modifier = Modifier.fillMaxWidth(),
+                        textAlign = TextAlign.End,
+                    )
+                },
+                isError = age.isNotEmpty() && (age.toIntOrNull()?.let { it < minAge || it > maxAge } ?: true)
+            )
+            ExposedDropdownMenuBox(
+                expanded = isExpanded,
+                onExpandedChange = {
+                    isExpanded = !isExpanded
+                },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 8.dp),
+            ) {
+                TextField(
+                    value = selectedOffice.toString(),
+                    onValueChange = {},
+                    readOnly = true,
+                    trailingIcon = { ExposedDropdownMenuDefaults.TrailingIcon(expanded = isExpanded) },
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .menuAnchor(),
+                )
+                ExposedDropdownMenu(
+                    expanded = isExpanded,
+                    onDismissRequest = { isExpanded = false },
+                    modifier = Modifier.fillMaxWidth(),
+                ) {
+                    viewModel.offices.forEach { item ->
+                        DropdownMenuItem(
+                            text = { Text(text = item.toString()) },
+                            onClick = {
+                                selectedOffice = item
+                                isExpanded = false
+                            }
+                        )
+                    }
+                }
+            }
+
+            Text(
+                text = stringResource(id = R.string.create_profile_screen_heading_section2),
+                modifier = Modifier
+                    .padding(top = 24.dp)
+                    .padding(bottom = 16.dp),
+                style = Typography.bodyLarge,
+                fontWeight = FontWeight.SemiBold,
+            )
+            OutlinedTextField(
+                value = firstTruth,
+                onValueChange = {
+                    if (it.length <= maxCharFacts) firstTruth = it
+                },
+                label = { Text(stringResource(id = R.string.create_profile_screen_truth1_label)) },
+                modifier = Modifier.fillMaxWidth(),
+                supportingText = {
+                    Text(
+                        text = "min $minCharFacts char., ${firstTruth.length} / $maxCharFacts",
+                        modifier = Modifier.fillMaxWidth(),
+                        textAlign = TextAlign.End,
+                    )
+                },
+                isError = firstTruth.isNotEmpty() && firstTruth.length < minCharFacts,
+            )
```

Comment:
This textfield is re-used in a very similar fashion, IMO they could/should be refactored to a separate composable to have less repetition.

Reply:
Done, good suggestion.